libamqpprox/amqpprox_maybesecuresocketadaptor.h

```diff
@@ -39,7 +41,7 @@ class MaybeSecureSocketAdaptor {
     using endpoint    = boost::asio::ip::tcp::endpoint;
     using handshake_type = boost::asio::ssl::stream_base::handshake_type;
 
-    boost::asio::io_service                               &d_ioService;
+    boost::asio::io_service &                              d_ioService;
```

Comment:
I think this will be fixed if the latest clang-format version is used for formatting.